test_data/issue-132.good.nix

```diff
@@ -1,5 +1,6 @@
 {
-  buildInputs = [ ] ++ stdenv.lib.optionals enableGui (with qt5; [ qtbase qtwebkit ])
+  buildInputs = [
+  ] ++ stdenv.lib.optionals enableGui (with qt5; [ qtbase qtwebkit ])
```

Comment:
why does `[ ]` get split into multiple lines here?

Reply:
it is because of the previous format, you can see in the `.bad.nix`. So, assuming that the developer put it in multiple lines in the first time, we don't wrap it.

So it is not inline that got expanded, but the other way around.

This not only gives developer freedom, but also give us more flexible formatter. That's why I change the `issue-132`  `test_data`